src/activity_item_handling.cpp

```diff
@@ -1093,23 +1095,27 @@ void activity_on_turn_move_loot( player_activity &, player &p )
                     }
                     // check free space at destination
                     if( free_space >= ( *it )->volume() ) {
-                        // before we move any item, check if player is at or adjacent to the loot source tile
+                        // before we move any item, check if player is at or
+                        // djacent to the loot source tile
```

Comment:
looks like you lost an "a" here, and an "s" in the next two comment changes

Reply:
Thanks, fixed.